target/arm/helper.c

```diff
@@ -14033,11 +14033,18 @@ void aarch64_sve_change_el(CPUARMState *env, int old_el,
 #endif
 
 #ifdef CONFIG_TCG_LOG_INSTR
-void HELPER(arm_log_instr)(CPUARMState *env, target_ulong pc, uint32_t opcode)
```

Comment:
Why the switch from `target_ulong` to `uint64_t`?  Is it to avoid having to call `tcg_temp_free`?  I guess there isn't a `tcg_constant_tl` you could use instead of `tcg_const_tl`?